packages/cli/src/services/jwt-base.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Service } from 'typedi';
+import * as jwt from 'jsonwebtoken';
+
+@Service()
+export class JwtBaseService {
```

Comment:
do we need this separate base service ?

Reply:
figured that if we need to sign/verify something in the feature we can extend from this one and inject the new secret. But, I'm also find keeping everything in one service! Thoughts?

Reply:
I'd prefer that we move this all to `jwt.service.ts` for now, and think about sub-classing when we actually need it.